views/user_registrations/new.haml

```diff
@@ -0,0 +1,11 @@
+.login-form
+  %form.form-signin{action: '/user_registrations/create', method: 'POST'}
+    =Rack::Csrf.tag(env)
+    %h2.form-signin-heading Please sign up
+    %label.sr-only{:for => "email"} Email address
+    %input.form-control{:autofocus => "", :name => "email", :placeholder => "Email address", :required => "", :type => "email"}/
+    %label.sr-only{:for => "password"} Password
+    %input.form-control{:name => "password", :placeholder => "Password", :required => "", :type => "password"}/
+    %label.sr-only{:for => "password_confirmation"} Password confirmation
+    %input.form-control{:name => "password_confirmation", :placeholder => "Password confirmation", :required => "", :type => "password"}/
```

Comment:
Line is too long. [137/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace